API.md

```diff
@@ -322,6 +322,14 @@ Queues a thumbnail regeneration task for the record given by `recordId`
 }
 ```
 
+### DELETE `/account/archive/:archiveId`
+
+Used by accounts that want to leave an archive that is shared with them.
```

Comment:
I would say "leave an archive of which they are a member", just to avoid conflating terms, because "sharing" is a separate concept in our application (it applies to records and folders, not to archives).